microsoft-edge/extensions/publish/marketing-guidelines.md

```diff
@@ -0,0 +1,121 @@
+
+Unlike generic Microsoft Store badges,<!-- todo: .png(s) --> the Microsoft Edge Add-on badge provides a distinct, trusted visual cue that connects your add-on directly to the Microsoft Edge ecosystem.
+<!-- todo: which?
+generic Microsoft Store badges - what's meant? show
+the Microsoft Store logo - show, if relevant here
+-->
+
+Use the provided badges in the `.zip` file (below), rather than creating your own image files.
+
+
+<!-- ====================================================================== -->
+## Minimum size for badges
+
+To ensure that the badge text remains readable, display it at a minimum size of 32px.  Avoid using smaller sizes.
+
+When scaling the badge, always maintain its aspect ratio; do not stretch or compress the badge image.  This helps preserve visual quality and reinforces your product's Credibility.<!-- todo: credibility? -->
+
+Follow the required specifications for image usage, fonts, and color palette.  For detailed usage examples and guidelines, see the [Microsoft Store Badge Guidelines](https://github.com/microsoft/MicrosoftEdge-Extensions/blob/main/assets/microsoft-store-badge-guidelines.pdf)<!-- todo: match dest title --> PDF file.
+
+
+<!-- ====================================================================== -->
+## Badge image files
+<!-- todo: Featured badge? Microsoft Edge Add-on badge? -->
+<!-- todo: mention "logo(s)"? -->
+
+Download the complete set of badge<!-- todo: Microsoft Edge Add-on badge?  Featured badge? --> `.png` image files: [microsoft-edge-add-ons-badges.zip](https://github.com/microsoft/MicrosoftEdge-Extensions/blob/main/assets/microsoft-edge-add-ons-badges.zip).
+
+These badges are available in many languages.
+
+
+<!-- ====================================================================== -->
+## Badge usage guidelines
+<!-- todo: what kind of badges?
+the Microsoft Edge Add-on badge
+the Featured badge
+-->
+
+Always use the official badge artwork provided by Microsoft.
+
+Do not create your own version, alter the text color, extract the Microsoft Edge logo, or modify the badge in any way.
+
+Do not animate, rotate, tilt, stretch, compress, or apply gradients or shadows to the badge.
+
+Do not use the badge as a decorative element in any context.
+
+The badge must always be an active, clickable link that directs users to your extension page at [Microsoft Edge Add-ons](https://microsoftedge.microsoft.com/addons/).
+
+Do not use the badge in static marketing materials, such as posters, flyers, or non-clickable images in presentations or videos.
+
+The badge is intended solely to facilitate direct access to your content on the Edge Add-ons platform.
+
+
+<!-- ====================================================================== -->
+## License for Microsoft Marks (badges and logos)
+
+_Microsoft Marks_ refers to the Microsoft Edge badges and logos<!-- todo: strike "and logos?"  this article is not at all about how to use a logo --> (assets) described above.
```

Comment:
strike "and logos?" this article is not at all about how to use a logo

@G-u-r-j-e-e-t